cli/cmd/install.go

```diff
@@ -54,6 +55,12 @@ This command will install k8s components that will auto-instrument your applicat
 		cmd.Flags().StringSliceVar(&ignoredNamespaces, "ignore-namespace", DefaultIgnoredNamespaces, "--ignore-namespace foo logging")
 		fmt.Printf("Installing Odigos version %s in namespace %s ...\n", versionFlag, ns)
 
+		nsflag, err := resources.GetOdigosNamespace(client, ctx)
```

Comment:
nit: this ns has nothing to do with any flag.
```suggestion
		existingOdigosNs, err := resources.GetOdigosNamespace(client, ctx)
```

Reply:
Changed name to `existingOdigosNs`